opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -265,6 +265,14 @@ class TraceOptions(int):
     def get_default(cls) -> "TraceOptions":
         return cls(cls.DEFAULT)
 
+    @classmethod
+    def get_sampled(
```

Comment:
This is a bit confusing, we have already a `sampled` attribute and now we are adding a `get_sampled` method which does not return the same thing as `sampled`.